src/views/SignTransactionSuccess.vue

```diff
@@ -1,12 +1,12 @@
 <template>
     <div class="container pad-bottom">
         <SmallPage>
-            <Success
-                text="Your transaction[br]is ready!"
-                buttonText="Send now"
-                :disabled="!isTxPrepared"
-                @continue="done"
-            />
+            <Loader state="success">
+                <template slot="success">
+                    <div class="success nq-icon"></div>
+                    <h1 class="title nq-h1">Sending your<br>transaction now.</h1>
```

Comment:
I'm not quite happy with stating it is sent now, while it isn't before returning to Safe.

Reply:
Especially with the new pico consensus, we could just as well send it here, just like during checkout.